lib/puppet/file_serving/configuration.rb

```diff
@@ -88,6 +89,8 @@ def mk_default_mounts
     @mounts["pluginfacts"].allow('*') if @mounts["pluginfacts"].empty?
     @mounts["tasks"] ||= Mount::Tasks.new("tasks")
     @mounts["tasks"].allow('*') if @mounts["tasks"].empty?
+    @mounts["plans"] ||= Mount::Plans.new("plans")
+    @mounts["plans"].allow('*') if @mounts["plans"].empty?
```

Comment:
What ended up requiring this to be here? We should never have to serve plan content

Reply:
This was included for feature parity with what we have for tasks. I will remove it.